### PR TITLE
Adding 'Atlas toolkit' as new tool.

### PR DESCRIPTION
**Pull request by epeios-q37**
# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
  - [x] includes a valid (https) link,
  - [x] includes a concise and on-topic description,
  - [x] mentions if there is only support some devices,
  - [x] is not a duplicate,
  - [x] has been in the correct alphabetical order for its section,
  - [x] is in the form of **Named Link - Description, seperated by commas.**
  - [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ The complete collection of (consumer) Raspberry Pi models consist of:
 
 - [Alpha](https://github.com/farjump/raspberry-pi) - Remotely load, debug and test bare-metal programs using GDB with this system-level GDB server.
 - [ApplePi Bakery](http://www.tweaking4all.com/software/macosx-software/macosx-apple-pi-baker/) - macOS application to easily install/backup/restore images onto an SD card.
+- [*Atlas* toolkit](https://github.com/epeios-q37/atlas-python) - Very lightweight and easy to install toolkit to write in *Python* (no *JavaScript* needed) web applications to pilot your RPi, without having to install a web server.
```
Comment (thibmaek) [inline README.md:97]: Couple of remarks:

- Don't use markup inside of the link or description.
- Skip the Javascript part, that's irrelevant to the short summary

**Review by thibmaek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (epeios-q37) [reply README.md]: The JS part was so the reader does not think that the Atlas toolkit is yet another framework to produce single-page application where the backend is written in Python, but the frontend in JS… (the Atlas toolkit is not for writing CGI-like application either).

**Comment by thibmaek** [comment]
🤝 Thanks for contributing this resource!

**Comment by JZavalaa** [comment]
I'll pass on this.   
  
  
  

  
  

  
  
>

**Comment by thibmaek** [comment]
> I'll pass on this.

@JZavalaa i am not sure what you mean? I don't think this PR relates to you in any way?

**Pull request merged**